web/assets/music.js

```diff
@@ -0,0 +1,31 @@
+import { MUSIC_ENABLED } from "../config.js";
+
+let playing = false
+
+export const startMusic = () => {
+  const music = document.getElementById("loadscreen-music");
+  if ( MUSIC_ENABLED ) {
+    playing = true
+    music.volume = 0.1;
+    document.addEventListener('keypress', (event) => {
+      if(event.key == "p") {
```

Comment:
Use `event.code` as its the most consistent across layouts and keyboards. If we are gonna allow for volume control using the arrow keys it might be conducive to provide user feedback in the form of an icon etc.

Reply:
Ok got it